docs/visual-basic/misc/bc42002.md

```diff
@@ -50,5 +50,5 @@ translation.priority.mt:
   
 ## See Also  
  [Classes for Visual Basic 6.0 Users](http://msdn.microsoft.com/en-us/d625222c-cd32-4c8d-b25c-ea71729b88b7)   
- [NOT IN BUILD: Using Constructors and Destructors](http://msdn.microsoft.com/en-us/548eebe1-86c4-4377-b2f5-447cb8be3d90)   
+ [Constructors](~/docs/visual-basic/programming-guide/concepts/object-oriented-programming.md#constructors)   
  [NOT IN BUILD: How to: Implement the Dispose Finalize Pattern (Visual Basic)](http://msdn.microsoft.com/en-us/adf7a232-4ebb-485d-8626-8d64421eb0c4)
```

Comment:
This can be replaced with `[Implementing a Dispose method](~docs/standard/garbage-collection/implementing-dispose.md)`